Entities/Characters/Scripts/RunnerDeath.as

```diff
@@ -90,7 +94,20 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 		this.getShape().getVars().onladder = false;
 		this.getShape().checkCollisionsAgain = true;
 		this.getShape().SetGravityScale(1.0f);
-		// fall out of attachments/seats // drop all held things
+		//set velocity to blob in hand
+        CBlob@ carried = this.getCarriedBlob();
+        if(carried !is null){
+            Vec2f current_vel = this.getVelocity() * CARRIED_BLOB_VEL_SCALE;
+            if(carried.hasTag("medium weight"))
+                current_vel = current_vel * MEDIUM_CARRIED_BLOB_VEL_SCALE;
```

Comment:
(afaik this should work with `*=` - not going to hold the PR just for that, but noting it for your benefit)